packages/sqs-receiver-service/src/show-queue-statistics.js

```diff
@@ -0,0 +1,52 @@
+import AWS from 'aws-sdk'
+import db from 'debug'
+
+AWS.config.update({ region: process.env.AWS_DEFAULT_REGION })
```

Comment:
Remove region specifier - this is not required as the AWS SDK will automatically use AWS_REGION (not AWS_DEFAULT_REGION) - see https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/setting-region.html